imports/ui/unit-explorer/unit-explorer.jsx

```diff
@@ -43,13 +50,17 @@ class UnitExplorer extends Component {
                 ))}
               </div>
             </div>
-            <Link to='/unit/new' className='link'>
-              <MenuItem className='no-shrink' innerDivStyle={{padding: 0}}>
-                <div className='tc bondi-blue fw5 bt b--moon-gray'>
-                  Add Unit
-                </div>
-              </MenuItem>
-            </Link>
+            {fabDescriptors.map((desc, ind) => (
+              <div key={ind} className='absolute bottom-2 right-2'>
+                <FloatingActionButton
+                  backgroundColor={desc.color}
+                  className='zoom-effect'
```

Comment:
You don't need 'zoom-effect' in this case. It was used to show the transition animation in the other place you found this,